### PR TITLE
Split Mock v3+ from v2 [v2]

### PR DESCRIPTION
**Pull request by praiskup**
Closes: #755

### CODE REVIEW

File: .tito/releasers.conf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [fedora-git-all]
 releaser = tito.release.FedoraGitReleaser
-branches = rawhide f35 f34 epel9 epel8 epel7
+branches = epel7
```
Comment (xsuchy) [inline .tito/releasers.conf:3]: you've got this in reverse. This should go to mock-2 and what you have in mock-2 should go to main.

Comment (praiskup) [reply .tito/releasers.conf:3]: Right, fixed.

**Comment by praiskup** [comment]
This needs a CI fix.  I will flip the CI from Travis to GH Actions or Packit first.

**Comment by xsuchy** [comment]
I recommend changing builder to tito.builder.UpstreamBuilder

**Comment by praiskup** [comment]
I know about that one though isn't it better to just wrap a new v2.X release, the normal way, than play with patches?

**Review by Conan-Kudo — APPROVED** [review]

**Review by xsuchy — APPROVED** [review]

**Pull request merged**